OBAKit/Controls/ListView/Helpers/OBAListViewDebug.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  OBAListViewPreview+CustomContent.swift
+//  OBAKit
+//
+//  Created by Alan Chu on 10/8/20.
+//
+
+import Foundation
+
+#if DEBUG
```

Comment:
please add a one sentence comment to this file to explain what the structs and classes in it are for.